src/components/Header/Header.js

```diff
@@ -1,35 +1,84 @@
 import React from 'react'
 import { IndexLink, Link } from 'react-router'
 import classes from './Header.scss'
-import SideMenu from '../SideMenu/SideMenu';
+import { connect } from 'react-redux'
 
-export const Header = () => {
+class Header extends React.Component {
 
+    triggerToggleCycleFlag = () => {
+      this.props.toggleCycleFlag()
+    }
+
+    triggerMenuShow = () => {
+      this.props.handleShowMenu()
+    }
+    componentWillReceiveProps(nextProps){
+      let cycleThrough = this.props.cycleThrough
+      let cropImageName = this.props.cropImageName
+      if(nextProps.cycleFlag === true){
+         cycleThrough(cropImageName)
+      }
+    }
+
+    render(){
+    let { cropImageName, cycleThrough, toggleCycleFlag, cycleFlag } = this.props
     return (
       <div>
-        <header>
+        <header >
           <div className="logo" style={{flex: 10}}>
             <img src="../../CC-logo.png" alt="CropCompass logo" width="200" height="100" />
+
+
           </div>
-          <nav>
+
+          <h2><a href={`wikipedia.com/${this.props.cropName ? this.props.cropName : ""}`}>
+              Current crop: {this.props.cropName ? this.props.cropName : ""}

```

Comment:
Also if you're going to do this in several spots I'd suggest putting a const cropName = this.props.cropName || "" at the top